### DISCUSSION TITLE
关于历届PCL版本的事情...

### DISCUSSION THREAD

**Discussion by kong-que-dnf123** [Meloong-Git/PCL#1857] [讨论]
今天来到旧电脑，猛然发现了2.3.0的老版本
所以有没有考虑过做个历代PCL版本的收录

（也可能是我没找到......如果真有收录的回一下那个网站吧谢谢）

——尝试上传  “2.3.0老版本（管他原来有什么东西呢 直接压缩下来得了）.zip”  ——
         ————压缩包文件过大，上传失败——————

**Comment by iPodtouch6557** [comment]
我还有alpha1.0.8╰( ´・ω・)つ

**Reply by iPodtouch6557** [reply]
> 我还有alpha1.0.8╰( ´・ω・)つ

哦对了，我还有release2.0.5和2.1.4

**Reply by MimiRabbit87** [reply]
我有Beta2和Release2.3.2

**Comment by so1ve** [comment]


**Reply by WForst-Breeze** [reply]
> 雀氏……要不然明明minecraft修了那么多bug，为啥我还不喜欢新版，愿意玩alpha1.2.3_04？bug一大堆不代表使用者用了有生命危险，你们推崇新版，极力避雷旧版其实有问题=。=

这位朋友本意的您应该是理解错了……并没有什么极力避雷，可能就是单纯的想问一下、表达一下观点而已😕

**Reply by iPodtouch6557** [reply]
> 有什么意义吗，体验古老bug…？

别吵了，我删了

**Reply by iPodtouch6557** [reply]
> 你觉得没必要不代表别人觉得没必要，理解他人的特殊需求是构建和谐社会非常重要的一部分

莫名觉得好笑（）

**Reply by Chiloven945** [reply]
锁一下吧

**Reply by so1ve** [reply]
可以了哥哥们，没必要在这里吵，有事去孙吧处理别天天引战也别挖坟

**Reply by WForst-Breeze** [reply]
已将此 Discussion 关闭，请各位尽量保证和谐、友善、理性、客观的讨论环境😀谢谢~！
